assayist/processor/utils.py

```diff
@@ -96,6 +96,13 @@ def download_build_data(build_identifier, output_dir='/metadata'):
     assert_command('koji')
     koji_session = get_koji_session()
     build = koji_session.getBuild(build_identifier)
+
+    state = build.get('state')
+    if state != 1:  # COMPLETE
```

Comment:
What about `koji.BUILD_STATES['COMPLETE']`?

Reply:
Ah, duh, we depend on Koji anyway. Will refactor. Thanks!